accepted/future-releases/enhanced-enums/feature-specification.md

```diff
@@ -245,13 +246,13 @@ The enum values can call the declared constructors, or the default unnamed zero-
 
 Enum instances are objects like any other object, and with this change they can implement interfaces and inherit members from mixins. The main difference between an `enum` declaration and a hand-written “equivalent class” using the enum pattern is that:
 
-- The `enum` types implement `Enum`. The `Enum` type is otherwise sealed against instantiation, so no other *objects* than enum entries can implement it. Abstract classes and mixins can implement or extend `Enum`, but unless they are implemented or mixed into an `enum` declaration, no objects can implement that type.
+- The `enum` types implement `Enum`. The `Enum` type is otherwise sealed against instantiation, so no other *objects* than enum entries can implement it. Abstract classes and mixins can implement or extend `Enum`, but unless they are then implemented by or mixed into an `enum` declaration, no objects can actually implement the type of that class or mixin.
 - The `enum` types themselves are completely sealed. No other class can implement an `enum` type.
 
 - Because of that, `enum` types support exhaustiveness checking in `switch` cases in the language _(meaning that flow-control can see that an exhaustive switch over enum values cannot pass through without executing at least one `case`, which can then affect variable promotion)_.
 - The `EnumName.name` extension member works on `enum` values.
 
-If the *restrictions* (the type is sealed, there is only a finite, enumerable number of instances, and the class  implements `Enum`, so it must have an `int index` getter and no `values` member), are acceptable, there should no longer be any reason to *not* make your enum-like class a language-based `enum`.
+If the *restrictions* (the type is sealed, there is only a finite, enumerable number of instances, and the class  implements `Enum`, so it must have an `int index` getter and no `values` member, and it cannot override `index`, `hashCode` or `operator==`), are acceptable, there should no longer be any reason to *not* make your enum-like class a language-based `enum`.
```

Comment:
> ...so it must have an `int index` getter and no `values` member, and it cannot override `index`...

Must have `int index` and cannot override `index` sounds conflicting.

Reply:
I think we can just avoid mentioning that `index` must be there (that's implied by the following 'cannot override `index`' anyway). With a couple of other adjustments:

```suggestion
If the *restrictions* are acceptable (the type is sealed; there is only a finite, statically known set of instances; the class  implements `Enum`; it cannot override `index`, `hashCode` or operator `==`; and it cannot have a user-written member named `values`), there should no longer be any reason to *not* make your enum-like class a language-based `enum`.
```